docker/linux-xenial-gcc5-tapir5.0/Dockerfile

```diff
@@ -4,6 +4,7 @@ FROM tensorcomprehensions/linux-xenial:${BUILD_ID}
 ENV DEBIAN_FRONTEND noninteractive
 
 RUN apt-get update
+RUN apt-get install -y --no-install-recommends vim
```

Comment:
Any reason why we're adding vim?

Reply:
yep, I normally have to build conda packages and sometimes have to edit the build scripts locally. Plus, users are running docker images as well so it's better to provide one editor at least :)